specops/localizedstrings/english.json

```diff
@@ -28,7 +28,6 @@
     "MENU_SP_LEVEL_AVG_TIME": "Est. Completion Time: ",
     "MENU_SP_LEVEL_YOUR_BEST_TIME": "Your Best Time: ",
     "MENU_SP_TYPE_TIMED_DRIVING": "Timed Driving",
-    "MENU_SP_SPECIAL_OPS_CAPS": "SPECIAL OPS",
```

Comment:
Why?

Reply:
This unnecessarily overwrites already translated string from built-in localizations.
![182247050-1da979e1-90b5-4b50-b155-b6b66fda94a0 (1)](https://user-images.githubusercontent.com/13555921/182247127-bc269cee-55b5-4851-946b-c8673602e106.png)